### ISSUE TITLE
Bug: The property handler mapping via FluentMapper or PropertyHandlerMapper is not being triggered.

### ISSUE THREAD

**Issue by mikependon** [mikependon/RepoDB#438]
When creating a property handler and you used either of the `FluentMapper.Entity<T>()` or `PropertyHandlerMapper.Add<TEntity, TPropertyHandler>()` , then the mapping is not being triggered.

**Property Handler:**

```csharp
public class StringToGuidPropertyHandler : IPropertyHandler<string, Guid>
{
	public Guid Get(string input, ClassProperty property)
	{
		var output = Guid.Empty;
		Guid.TryParse(input, out output);
		return output;
	}

	public string Set(Guid input, ClassProperty property)
	{
		return input.ToString();
	}
}
```

**Class Model:**

```csharp
public class Customer
{
	public Guid Id { get; set; }
	public string Name { get; set; }
}
```

**Mapping:**

```csharp
FluentMapper.Entity<Customer>()
	.PropertyHandler<StringToGuidPropertyHandler>(e => e.Id);
```

Upon investigation, the issue is on the compiler level and that needs to be fixed there.

**Comment by mikependon** [comment]
This will be available on RepoDb release (>= v1.11.1-beta2).

**Comment by mikependon** [comment]
The fix is now available in version RepoDb v1.11.1.
